### ISSUE TITLE
3 pices Aquara RTCGQ11LM lose connection over time and go offline

### ISSUE THREAD

**Issue by richimaint** [Koenkk/zigbee2mqtt#13067]
### What happened?

My 3 Aqara motion sensors RTCGQ11LM lose the connection over time and go offline. Then I can either press the button on the devices and the connection is eventually restored or I have to pair them again.
I have ZigBee2MQTT only since version 1.25.0

### Zigbee2MQTT version

1.25.2

### Adapter firmware version

20220219

### Adapter

Sonoff Zigbee 3.0 USB Dongle Plus,TI CC2652P +

**Comment by richimaint** [comment]
![image](https://user-images.githubusercontent.com/91455484/177761237-445ef7ff-426c-4c59-bce4-8479a2b9c362.png)

**Comment by Koenkk** [comment]
Make sure to check https://www.zigbee2mqtt.io/devices/RTCGQ11LM.html#troubleshooting-device-stops-sending-messages-disconnects-from-network

**Comment by julianfs** [comment]
I concur with this issue, I have seen the same on 2 of my 4 devices.

**Comment by richimaint** [comment]
Incorrect LQI values are displayed in the overview. In the map, most devices are connected to my router.

![image](https://user-images.githubusercontent.com/91455484/177985754-de01f82e-832e-415b-8212-b071175f6e09.png)

**Comment by richimaint** [comment]
I set the occupancy_timeout to 10 and now it has been running for 3 days without losing the network connection.
Maybe it was the solution.

![image](https://user-images.githubusercontent.com/91455484/178510774-b0e43dfa-2669-456f-a4eb-ed1c021f8688.png)

**Comment by richimaint** [comment]
the problem still exists, but it takes longer until the next time

**Comment by oystermon** [comment]
Hi richimaint,

I don't think the LQI values in the map are incorrect. The motion sensor to your dongle LQI is strong, but the LQI to your coordinator (the star) is very weak (35). To me that's the main issue with your current setup..

I think Koenkk has already provided good information that you should look through properly, but if you check on google and look for z2m maps, you will see they look very different to your setup:
http://flemmingss.com/wp-content/uploads/2021/09/zigbee2mqtt_setup_gui_map.png
https://libreddit.nl/img/r4kajezklml81.png

**Comment by richimaint** [comment]
i have a new position of Sonoff Router and the LQI is better:

**Comment by richimaint** [comment]
I still have the same problem.

richimaint

**Comment by ilgio** [comment]
I have same problem, I bought also lamp ikea, but after 3 month the sensor is disconnected

**Comment by movy** [comment]
Same problem with 2 Aqara devices. LQI is irrelevant I believe, as even if I position devices within 2-3 meters from the receiver (i.e. LQI > 100) both devices stop reporting occupancy, but still report luminance changes and they do instantly report status upon  'pair button" press.

**Comment by movy** [comment]
So, just to make sure, I have updated Sonoff firmware and Z2M to latest versions, no avail. Upon further mingling it turns out the device reports status if you squeeze it a little in your hand (i.e. press top and bottom sides with your fingers). One might think it's a battery issue, but I have replaced battery couple times with brand new one, voltage shows 3205, mended contacts to make sure they touch the battery -- still have to squeeze to make it work. Go figure, might be some soldering issue inside. Overall, from my experience with number of brands, Phillips HUE devices work the best (they also begin to get wonky when battery begins to die, but that's about it and it is predictable).

**Comment by Flux3PO** [comment]
One side note for me. I was strugeling with this too. I got like 10 RTCGQ11LM and most of them are fine while others refuse ti play ball. So I did found thisa posting after reading some others and concluded the following.
I only got 10 motion sensors and 3 water leak sensors from Aqara/Xiaomi and like 10 Ikea devices. I usually added them to the network by using the Permit Join (All) option. I read somewhere that the light bulps are not good in transfering the Xiaomi Messages as they are none standard. 

So, I paired the motion sensors with my Ikea outlets and this seems to do the trick for like 48h without any issue since then.

Not sure if this is worth to be added to the troubleshooting section @Koenkk  ?
